darwin/future/meta/client.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+from typing import Optional
+
+from requests.adapters import Retry
+
+from darwin.future.core.client import Client, DarwinConfig
+
+
+class MetaClient(Client):
+    def __init__(self, config: DarwinConfig, retries: Optional[Retry] = None) -> None:
+        super().__init__(config, retries=retries)
+
+    @classmethod
+    def local(cls) -> Client:
+        config = DarwinConfig.local()
+        return MetaClient(config)
+
+    @classmethod
+    def from_api_key(cls, api_key: str, datasets_dir: Optional[Path] = None) -> Client:
```

Comment:
Function sig says this returns client, but it returns `MetaClient` - which obv is a subclass anyway, but any reason not to just say `MetaClient` as a return?

Reply:
good catch, I extracted this out from the core client so that's why.